src/components/user/LastUserWidget.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <p class="title text-lg-center">
+      Users
+    </p>
+    <v-text-field
+      label="Search"
+      @input="(q)=>fetchUsers({q})"
+    />
+    <div v-if="loading" class="text-xs-center mt-5">
+      <v-progress-circular
+        :width="1"
+        color="red"
+        :size="70"
+        indeterminate
+      />
+    </div>
+    <div v-for="user in userList" v-else :key="user.id" class="avatars">
```

Comment:
Please fix problem with `v-else` here too

Reply:
Wrapped the v-else div in a template, as well as the loader div, so that they are wrapped in the same tag